spec/unit/multi_select_spec.rb

```diff
@@ -905,4 +909,48 @@ def exit_message(prompt, choices)
       expect(prompt.output.string).to eq(expected_output)
     end
   end
+
+  context "with :min and :max" do
+    let(:choices) { %w[A B C] }
+    before do
+      prompt.on(:keypress) do |e|
+        prompt.trigger(:keydown) if e.value == "j"
+      end
+    end
+    context "with min <= max" do
+      let(:min) { 2 }
+      let(:max) { 2 }
+      it "requires a min number of choices" do
+        keydown = "j"
+        spacebar = " "
+        enter = "\r"
+
+        prompt.input << spacebar << enter << keydown << spacebar << enter
+        prompt.input.rewind
+
+        value = prompt.multi_select("What letter?", choices, min: min, max: max, per_page: choices.count)
+        expect(value).to eq(%w[A B])
+
+        expected_output =
+          output_helper("What letter?", choices, "A", [], init: true, min: min, max: max,
+            hint: "Press #{up_down} arrow to move, Space to select and Enter to finish") +
+          output_helper("What letter?", choices, "A", %w[A], min: min, max: max) +
+          output_helper("What letter?", choices, "A", %w[A], min: min, max: max) +
+          output_helper("What letter?", choices, "B", %w[A], min: min, max: max) +
+          output_helper("What letter?", choices, "B", %w[A B], min: min, max: max) +
+          exit_message("What letter?", %w[A B])
+
+        expect(prompt.output.string).to eq(expected_output)
+      end
+    end
+    context "with min > max" do
+      let(:min) { 3 }
+      let(:max) { 2 }
```

Comment:
Unnecessary lets, in general, I'm not a fan of `lets`, use them sporadically. Use the integers directly in the test body without any variable assignment.